packages/server/src/login/login.controller.ts

```diff
@@ -165,4 +165,18 @@ export class LoginController {
       .cookie('auth_token', authToken, { httpOnly: true, secure: isSecure })
       .redirect(302, '/');
   }
+
+  @Get('/logout')
+  async logout(@Res() res: Response): Promise<void> {
+    const isSecure = this.configService
+      .get<string>('DOMAIN')
+      .startsWith('https://');
+    res
+      .cookie('auth_token', '', {
```

Comment:
Would it make sense to use `res.clearCookie` here?

https://expressjs.com/en/4x/api.html#res.clearCookie

Reply:
yeah probably